.claude/skills/code/SKILL.md

```
@@ -74,3 +74,13 @@ Format:
 ```
 
 See `.github/CONTRIBUTING.md` for full conventions. Pre-commit hooks run on commit — fix any failures and re-commit.
+
+## Step 5: Continue — invoke /pr immediately
+
+**Do not stop after committing.** Invoke the `/pr` skill immediately to create the pull request. The full workflow is a single uninterrupted pipeline:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)